docs/en/02_Developer_Guides/01_Templates/10_Unique_Keys.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Generating Unique Keys
+summary: Outputting unique keys in templates.
+icon: code
+---
+
+# Unique Keys
+
+There are several cases where you may want to generate a unique key. For example:
+
+* populate `ID` attribute in your HTML output
+* key for partial cache
```

Comment:
I'd note that these two use-cases have different constraints. Populating an ID requires more consideration of human factors (such as maybe not dumping in the FQCN) that you don't need to worry about when generating a key for a partial cache.

It's possible we can get a works-for-all-cases option, but that's not yet clear.